index.html

```diff
@@ -5,12 +5,34 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
     <title>API Practice</title>
 </head>
 
 <body>
-    <div class="container"></div>
-    <button id="random">Get Info</button>
+
+    <div class="container">
+        <P id="Name1"></P>
```

Comment:
a better way to do this is to make your JS do this part automatically for every new element, by using createElement()